### PR TITLE
text changes

### PR DESCRIPTION
**Pull request by afzaledx**

### CODE REVIEW

File: edx_proctoring/templates/proctored_exam/ready_to_start.html [hunk 1]
```diff
@@ -8,10 +8,10 @@
     </h3>
     <p>
       {% blocktrans %}
-        &#8226; Do not close the proctoring window until you have completed and submitted your exam. </br>
         &#8226; When you start your exam you will have {{ total_time }} to complete it. </br>
-        &#8226; If the allotted time expires before your end your exam, the answers you have completed up
-        to that point are submitted for grading and your proctoring session is uploaded for review. </br>
+        &#8226; You cannot stop the timer once you start. </br>
+        &#8226; If the time expires before you finish your exam, your completed answers will be
```
Comment (catong) [inline edx_proctoring/templates/proctored_exam/ready_to_start.html]: Strike "the" before "time": "If time expires before you finish..."

File: edx_proctoring/templates/proctored_exam/rejected.html [hunk 2]
```diff
@@ -6,16 +6,10 @@
...
-      Your proctoring session results: <b class="failure"> Unsatisfactory </b>
-    {% endblocktrans %}
-  </h4>
-
   <p>
     {% blocktrans %}
-      You are not eligible to purchase academic credit for this course, regardless of your final grade in the course.
-      If you have concerns about your proctoring session results, contact your course team.
+      You are no longer eligible for academic credit for this course, regardless of your final grade.
+      If you have questions about the status of your proctored exam results, please contact {{ platform_name }} Support.
```
Comment (catong) [inline edx_proctoring/templates/proctored_exam/rejected.html]: Strike "please": "...contact ```{{ platform_name }}``` Support."

File: edx_proctoring/templates/proctored_exam/submitted.html [hunk 3]
```diff
@@ -5,26 +5,23 @@
...
-      Your proctoring session results: <b> Pending </b>
-    {% endblocktrans %}
-  </h4>
 
   <p>
     {% blocktrans %}
-      After you have ended the proctoring session, the recorded data is uploaded for review. Proctoring session results are usually available 24-48 hours after you submit your exam.
-      If you have questions about the status of your session review
-      after that time, contact <a href="{{link_urls.contact_us}}" target="_blank">{{platform_name}}</a> Support.
+      If you have questions about the status of your proctored exam results, please contact {{ platform_name }} Support.
```
Comment (catong) [inline edx_proctoring/templates/proctored_exam/submitted.html]: Strike "please".

File: edx_proctoring/templates/proctored_exam/exam_not_submitted.html [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+  <p>
+    {% blocktrans %}
+      Make sure you return to the proctoring software and select <strong> Quit </strong> to end proctoring
+      and submit your exam.
+    {% endblocktrans %}
+  </p>
+
+  <p>
+    {% blocktrans %}
+      If you have questions about the status of your proctored exam results, please contact {{ platform_name }} Support.
```
Comment (catong) [inline edx_proctoring/templates/proctored_exam/exam_not_submitted.html]: Strike "please": 
"...status of your proctored exam results, contact ``{{ platform_name }} Support."

**Comment by afzaledx** [comment]
@chrisndodge, this story has all the stories of PHX-130 except for the PHX-149, which we're doing in another branch.

**Comment by chrisndodge** [comment]
Thank you. I'm a bit backlogged regarding this week's release. I hope to catch up on PR's later today.

File: edx_proctoring/templates/proctored_exam/submitted.html [hunk 5]
```diff
@@ -5,26 +5,23 @@
       You have submitted this proctored exam for review
     {% endblocktrans %}
   </h3>
+
+
```
Comment (muhhshoaib) [inline edx_proctoring/templates/proctored_exam/submitted.html]: Remove the extra line

**Comment by muhhshoaib** [comment]
Looks good to me +1

**Comment by chrisndodge** [comment]
@afzaledx can you squash commits and merge? Thanks.

**Pull request merged**